### PR TITLE
react 22 testing library

### PR DESCRIPTION
**Pull request by davelosert**
- react-22: Add initial code
- Adds first component tests

### CODE REVIEW

File: 22-testing-library/src/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <title>State-Testing mit Mocha</title>
```
Comment (goloroden) [inline 22-testing-library/src/index.html:5]: Should this be adjusted to something else, or is this fine?

**Review by goloroden — CHANGES_REQUESTED** [review]
Hi @davelosert 👋 

Thanks for the PR! I've got two questions on this:

1. I have left a comment on one file, could you please have a look at this?
2. It seems to me as if there are no tests yet, am I missing something here?

File: 22-testing-library/src/index.html (hunk 1 above)
Comment (davelosert) [reply 22-testing-library/src/index.html:5]: Hi @goloroden: Sorry, this is still a draft - still working on it. Fill fix the comment and assign the PR to you as soon as I am ready. 🙂

Comment (goloroden) [reply 22-testing-library/src/index.html:5]: Aaaaahhh, I missed that this was still a draft 😊

**Comment by davelosert** [comment]
@goloroden : I am out of draft state and ready for review.
I will deliver the slides probably tomorrow.

**Review by goloroden — APPROVED** [review]

**Pull request merged**